atd-vzd/views/view_crashes_inconsistent_numbers.sql

```diff
@@ -37,7 +37,7 @@ FROM atd_txdot_crashes atc
          LEFT JOIN primarypersons atpp ON atc.crash_id = atpp.crash_id
          LEFT JOIN units atu ON atc.crash_id = atu.crash_id
 WHERE 1 = 1
-  AND (austin_full_purpose = 'Y' OR (atc.city_id = 22 and atc.position IS NULL))
+  AND (atc.in_austin_full_purpose = true)
```

Comment:
just released that in the staging/prod DBs, this view doesnt actually even use `austin_full_purpose` in the where clause, its only here in the code that it does :sweat_smile: it instead just filteres by city_id = 22

![image](https://github.com/cityofaustin/atd-vz-data/assets/67130963/875876b2-3e45-4217-9186-5c6b4fbe9ab7)

these are the problems that come along with not having a great way to source control our db changes. does anyone have any context on this view/if its important, and should i just leave the view as is then?

Reply:
I have zero context on this view, but this is really frustrating -- i wonder how many other places the db doesnt match what we have in github

Reply:
The other views that i updated to use `in_austin_full_purpose` actually do use the austin full purpose flag, its just this one that doesnt match. but yeah curious where else this happens in github :face_with_spiral_eyes:

Reply:
this is frustrating! I think it is a combination of the criteria of AFP changing over time and being set in so many individual places that it probably wasn't consistently updated. This really highlights the upside of centralizing that logic in the new column that you've added in this PR! 🚀